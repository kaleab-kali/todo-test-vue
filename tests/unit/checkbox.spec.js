import Checkbox from "@/components/atoms/CheckBox.vue";
import { mount } from "@vue/test-utils";

describe("Atom Checkbox Component", () => {
  it("Correct layout", () => {
    const wrapper = mount(Checkbox);
    expect(wrapper.html()).toMatchSnapshot();
  });
  it("is clicked", async () => {
    const wrapper = mount(Checkbox);
    await wrapper.find("input").trigger("click");
    expect(wrapper.emitted().click).toBeTruthy();
  });
});
