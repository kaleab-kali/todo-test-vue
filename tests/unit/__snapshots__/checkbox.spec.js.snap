// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom Checkbox Component Correct layout 1`] = `<label><input type="checkbox" class="checkbox"> <span class="fake"></span></label>`;
